src/reflect/scala/reflect/internal/util/Collections.scala

```diff
@@ -55,6 +55,17 @@ trait Collections {
   final def mforeach[A](xss: List[List[A]])(f: A => Unit) = xss foreach (_ foreach f)
   final def mforeach[A](xss: Traversable[Traversable[A]])(f: A => Unit) = xss foreach (_ foreach f)
 
+  final def lastSuch[A](xs: List[A])(pred: A => Boolean): Option[A] = {
+    var res: A = null
```

Comment:
My bad; `null.asInstanceOf[A]`, that old chestnut.

Reply:
I am not used to using `null`, so mine as well  :¬)>~